### PR TITLE
Fix authorization

### PR DESCRIPTION
**Pull request by SavStanis**
Added validation into the AuthController

### CODE REVIEW

File: src/main/java/com/ncedu/cheetahtest/exception/validation/ValidationExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.util.stream.Collectors;
+
+@ControllerAdvice
+public class ValidationExceptionHandler extends ResponseEntityExceptionHandler {
+    @Override
+    protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotValidException ex,
+                                                                  HttpHeaders headers,
+                                                                  HttpStatus status, WebRequest request) {
+        Map<String, Object> body = new LinkedHashMap<>();
+        body.put("timestamp", new Date());
```
Comment (henchamax) [inline src/main/java/com/ncedu/cheetahtest/exception/validation/ValidationExceptionHandler.java]: How will you use "timestamp" ? Timezone is needed ?

File: .travis.yml [hunk 2]
```diff
@@ -20,7 +20,7 @@ script:
...
+    secure: Blbvoh7wsvcwIMuZYd+kkcWSItvMXDfAP+NGRZ5RSvZmaVaMQGUHQ7bw3tP8YqZ8jrDTzCg0BGfyEAzsFKiHTx5gqi1c9d/Vlb5NnlFb4yVYxnJh7rKz5mVKPsCHdM4dIHOI41zxaDJOMBv5GXUcoEbUYvxnDXcXL/Z1BPGCjCd9WkO2MkdW4TkICyalIbp/vUGaPM4rmOy4oGnJBW78wwTXoDAGl4PRjFvkbQUqnhd5mzJ4Ay6osdnmXbAG//iB0eVjuYW7b82Bi+5Jjnk4alur0MqrXumvwxL3b+CkyJbkyMnWlK0YI2XiGy21ZjM1iD335gVqVsBfaU0mvgfvvOjIF2X/Bdtw1+fQd+k/flSk0fVby9P35NPF3bYaAF+h5axAV7QCKOSvvkHiDVal6FnSwCdlQDKsnpdqojnVyqcc9ggM2M9Zna3m60iE0nZR6yPfuHLEg6PnKP8qdWMY6NilbWuD6PuUFrP2TXH774bku/ON0T/l1qrR+R81LdBTfdVeZKCWfubbjdXpxK98BtwIzg2UqDPEDpAID/1mnKIcYiuFxdHYrVkTLAH8Gi6clTHJbgIGoaEkqPb9aUdsZdrzPOJ2x1Wz/50sfxfVG/UyWb0L8gHQxr4xMl17kKoLhFrVcw15yKBUEGtHEFOqsnEZmn3zT1gYIydamqIMMeE=
```
Comment (henchamax) [inline .travis.yml]: Please, move to travis environment variables

**Review by HellaxA — COMMENTED** [review]
Checked

**Review by KotyaraSingleCat — APPROVED** [review]

**Comment by GorovDimon11** [comment]
Viewed

**Pull request merged**